cpp_fibonacci.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+    int i, n, t1 = 0, t2 = 1, t3;

```

Comment:
Codacy found an issue: [The scope of the variable 't3' can be reduced.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6329869)